Ladybird/AppKit/UI/TabController.mm

```diff
@@ -82,7 +82,7 @@ @implementation TabController
 @synthesize new_tab_toolbar_item = _new_tab_toolbar_item;
 @synthesize tab_overview_toolbar_item = _tab_overview_toolbar_item;
 
-- (instancetype)init
+- (instancetype)init:(BOOL)block_popups
```

Comment:
For our future selves, it probably makes more sense to pass in a `TabSettings` instead of an individual param for each setting.